src/Runner.Worker/ContainerOperationProvider.cs

```diff
@@ -98,12 +98,12 @@ public async Task StartContainersAsync(IExecutionContext executionContext, objec
                 await StartContainerAsync(executionContext, container);
             }
 
-            executionContext.Output("##[group]Waiting for all services to be ready");
```

Comment:
Without this `##[group]`, the log might be flooded with a bunch of `xxxx service is healthy.` in the health check pass scenario.